redash/utils/__init__.py

```diff
@@ -17,6 +18,7 @@
 from redash import settings
 
 COMMENTS_REGEX = re.compile("/\*.*?\*/")
+WRITER_ENCODE = os.environ.get('REDASH_WRITER_ENCODE', 'utf-8')
```

Comment:
Can you please change this to `WRITER_ENCODING` and `REDASH_CSV_WRITER_ENCODING`? Thanks.